In [5]:
input("tell me your name")

tell me your namefarhat


'farhat'

In [6]:
f_n = input("enter first number")
s_n = input("enter second number")

enter first number34
enter second number56


In [7]:
print(f_n)
print(s_n)

34
56


In [9]:
output = (f_n + s_n)
output

'3456'

property of input function = when ever user is giving input from "input function" that input function comes like string function to the programmer...'3456' here why python take as string becuse string is a universal data type which hold other data type


In [11]:
type(f_n)

str

# Type conversion

1. Implicit- when python do automatically type converion programmer dont need to specifiy
2. Explicit- when programer have to tell python on this point do the type conversion

In [12]:
int(4)

4

In [13]:
bool(0)

False

In [14]:
complex(9)

(9+0j)

In [16]:
list("hello")

['h', 'e', 'l', 'l', 'o']

In [18]:
f_n = input("enter first number")
s_n = input("enter second number")

result = int(f_n )+ int(s_n)
print(result)

enter first number23
enter second number10
33
